vxfreeswitch/voice.py

```diff
@@ -92,25 +94,25 @@ def create_and_stream_text_as_speech(self, folder, command, ext, message):
         else:
             log.msg("Using cached voice file %r" % (filename,))
 
-        yield self.playback(filename)
+        yield self.output_stream(filename, settings)
 
     @inlineCallbacks
-    def send_text_as_speech(self, engine, voice, message):
+    def send_text_as_speech(self, engine, voice, message, settings={}):
         yield self.set("tts_engine=" + engine)
         yield self.set("tts_voice=" + voice)
-        yield self.execute("speak", message)
+        yield self.output_stream("say:'%s'" % message, settings)
```

Comment:
The FreeSwitch docs (https://wiki.freeswitch.org/wiki/Misc._Dialplan_Tools_say) are making me worry that `say` is rather different to `speak`. Is there a difference in practice? If so, perhaps it's possible to use `speak` here anyway? Not sure.